README.md

```diff
@@ -261,17 +271,17 @@ sourceFile2.py.zip
 sourceFolder2.zip
 ```
 
-*NB:* When zipping a folder, the root of the archive will contain the contents
-of that folder, not the folder by itself.
+Both the `hash` and `zip` flag can be used together. This will yield the same
+result as having just the `zip` flag, except all file paths will be
+prefixed with the hash of their contents.
+
+*NB:* When zipping a folder, the structure of the folder is flattened since AWS
+Lambda can only detect source files in the root of an archive.
```

Comment:
That's good.

But even if lambda only supported the source entrypoint in the root, you'd still want to keep folders for other files in the zip. For example, you might want to zip and upload the contents of the lambda folder:

```
- lambda\
  - modules\
    - dependency.js
  - entrypoint.js
- evaporate.yaml
```
and it should be zipped into a zip file containing:
```
- modules\
  - dependency.js
- entrypoint.js
```